README.md

```diff
@@ -104,30 +115,40 @@ Implementation is very easy and can be performed in just a few steps (assuming t
 * place dedicated Twig helper in place where you want to display recommendations:
 
 ```twig
-{{ yc_show_recommendations(content.id, 'scenario_name', limit, 'content_type', 'template',
-    ['ez_publishedDate', 'ez_url', 'title', 'image', 'author', 'intro']
+{{ yc_show_recommendations(content.id,
+    scenario = '',
+    limit = '',
+    contentType = '',
+    template = '',
+    attributes = []
 ) }}
 ```
 
-Parameter meanings:
+Parameter meanings (all bellow are required):
 
 Parameter       | Type   | Description
 ----------------|--------|------------
 `content.id`    | int    | this is in content based views normally the twig variable holding the content id (we want to get recommendations for)
-`scenario_name` | string | scenario used to display recommendations, you can create one at YooChoose dashboard
+`scenario`      | string | scenario used to display recommendations, you can create one at YooChoose dashboard
 `limit`         | int    | how many recommendations will be shown?
-`content_type`  | string | content type values you are expecting in response
+`contentType`   | string | content type values you are expecting in response
 `template`      | string | HandleBars template name (your templates are stored under `EzRecommendationBundle/Resources/public/views` directory. Take a look on `default.html.twig` file which includes default template that can be used to prepare customised version)
-`[fields]`      | array  | here you can define which fields are required and will be requested from the recommender engine. These field names are also used inside HandleBars templates
+`attributes`    | array  | here you can define which fields are required and will be requested from the recommender engine. These field names are also used inside HandleBars templates
```

Comment:
Why did you change `fields` to `attributes` ? The latter isn't part of our Domain Language, while field is.